src/geo/cartodb-layer-group.js

```diff
@@ -46,15 +46,19 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     return !!this.get('urls');
   },
 
-  getTileURLTemplates: function () {
+  getTileURLTemplates: function (type) {
+    type = type || 'png';
+
     if (this._areAllLayersHidden()) {
       return [];
     }
     var tileURLTemplates = (this.get('urls') && this.get('urls').tiles) || [];
-    return _.chain(tileURLTemplates)
-      .map(this._replaceLayerIndexesOfVisibleLayers.bind(this))
+    var urls = _.chain(tileURLTemplates)
+      .map(this._replaceLayerIndexesOfVisibleLayers.bind(this, type))
       .map(this._appendAuthParamsToURL.bind(this))
       .value();
+
+    return type === 'mvt' ? urls && urls[0] : urls;
```

Comment:
Because Tangram doesn't need an array of tiles URLs, it's always the same URL as a source and it controls internally the tile requests.